mne_bids/tests/test_mne_bids.py

```diff
@@ -130,3 +130,75 @@ def test_bti():
     cmd = ['bids-validator', output_path]
     run_subprocess(cmd, shell=shell)
     assert op.exists(op.join(output_path, 'participants.tsv'))
+
+
+# EEG Tests
+# ---------
+def test_vhdr():
+    """Test raw_to_bids conversion for BrainVision data."""
+    output_path = _TempDir()
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                task=task, raw_file=raw_fname, output_path=output_path,
+                overwrite=True, kind='eeg')
+
+    cmd = ['bids-validator', '--bep006', output_path]
+    run_subprocess(cmd, shell=shell)
+
+
+def test_edf():
+    """Test raw_to_bids conversion for European Data Format data."""
+    output_path = _TempDir()
+    data_path = op.join(testing.data_path(), 'EDF')
+    raw_fname = op.join(data_path, 'test_reduced.edf')
+
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                task=task, raw_file=raw_fname, output_path=output_path,
+                overwrite=True, kind='eeg')
+
+    cmd = ['bids-validator', '--bep006', output_path]
+    run_subprocess(cmd, shell=shell)
+
+
+def test_bdf():
+    """Test raw_to_bids conversion for Biosemi data."""
+    output_path = _TempDir()
+    data_path = op.join(testing.data_path(), 'BDF')
+    raw_fname = op.join(data_path, 'test_bdf_stim_channel.bdf')
+
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                task=task, raw_file=raw_fname, output_path=output_path,
+                overwrite=True, kind='eeg')
+
+    cmd = ['bids-validator', '--bep006', output_path]
+    run_subprocess(cmd, shell=shell)
+
+
+def test_set():
+    """Test raw_to_bids conversion for EEGLAB data."""
```

Comment:
This will only pass on master of MNE-Python, once https://github.com/mne-tools/mne-python/pull/5504#issuecomment-420041448 is merged